src/server/api/routers/subscription.test.ts

```
@@ -0,0 +1,321 @@
+
+// ---------------------------------------------------------------------------
+// Hoisted mocks — must be before any imports
+// ---------------------------------------------------------------------------
+
+const mockDb = vi.hoisted(() => {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const db: any = {
+    user: {
+      findUnique: vi.fn(),
+    },
+    subscription: {
```

ESLint

[error] Expected object keys to be in natural insensitive ascending order. 'subscription' should be before 'user'.

(sort-keys-fix/sort-keys-fix)